wdl/ClusterBatch.wdl

```diff
@@ -65,6 +65,9 @@ workflow ClusterBatch {
 
     Float? java_mem_fraction
 
+    # PlotSVCountsPerSample metrics
+    Int N_IQR_cutoff_plotting = 6
```

Comment:
There was a previous draft PR that did part of this and I'm going to refer to a [comment](https://github.com/broadinstitute/gatk-sv/pull/389#discussion_r988160846) from that PR from Mark: 

> I'd suggest making this an optional. If it's provided, we run PlotSVCountsPerSample. If not, it skips the task (we don't need to run this for the single sample pipeline for example). Would need to add this input and the outputs to GATKSVPipelinePhase1 and GATKSVPipelineBatch. 

I didn't consider this when writing up the git issue so that's on me! I think it's ok for the plotting to run by default for FilterBatchSites (not optionally, because FilterBatchSites is not called in the single-sample WDL), but if you want you can make that optional in the same way as ClusterBatch. So this change would involve:
* Make the N_IQR input optional in ClusterBatch and remove the default of 6 in the WDL
* Add this input to GATKSVPipelinePhase1 and GATKSVPipelineBatch (which calls ClusterBatch) and pass it to the necessary subworkflows (no need to add it to the single-sample WDL as the plots aren't necessary there)
* Add this input to the JSON templates for GATKSVPipelinePhase1 and GATKSVPipelineBatch with a default value of 6
* Check if the N_IQR input is defined and only call the plotting subworkflow if it is
* Make the corresponding outputs optional since they won't be generated every time
* Re-validate all workflows and re-test ClusterBatch with and without the N_IQR input (and FilterBatchSites if you choose to make these modifications there) and make sure that the plots are generated or not generated as expected